### PR TITLE
Minor documentation typos

### PR DESCRIPTION
**Pull request by Zaituny**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -2,5 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ivy_tests/array_api_testing/test_array_api" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ivy_tests/array_api_testing/test_array_api/array-api" vcs="Git" />
```
Comment (xoiga123) [inline .idea/vcs.xml:6]: Please revert this change.

**Review by xoiga123 — CHANGES_REQUESTED** [review]
Thank you for contributing :hugs: However, please don't edit any files in `.idea`

**Comment by Zaituny** [comment]
> Thank you for contributing 🤗 However, please don't edit any files in `.idea`

My bad, pushed the changes by mistake 😅

**Review by xoiga123 — APPROVED** [review]
LGTM :hugs:
